wcivf/apps/core/context_processors.py

```diff
@@ -2,9 +2,15 @@
 
 from .forms import PostcodeLookupForm
 
+# Remove this once we add all_settings to settings.CONTEXT_PROCESSORS
 def canonical_url(request):
     return {'CANONICAL_URL': settings.CANONICAL_URL}
 
+
+def all_settings(request):
+    return {x: getattr(s, x) for x in dir(s)}
```

Comment:
Oh, I see, there are subsequent commits that have fixes for this. Never mind. That's the risk of of reviewing commit-by-commit, I guess....